vm/Cargo.toml

```diff
@@ -24,6 +24,7 @@ sha2 = "0.8"
 sha3 = "0.8"
 blake2 = "0.8"
 
+gethostname = "0.2.0"
```

Comment:
This should not be included in WASM. You can look #1201 for example.

Reply:
Thanks for your guidance, it helps a lot.